db/schema.rb

```diff
@@ -11,11 +11,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170325205929) do
 
+ActiveRecord::Schema.define(version: 20170326222205) do
```

Comment:
I think this might've gotten messed up in your merge, but I think the blank line is supposed to be _after_ the `ActiveRecord::Schema.define` line 🤔 

(not a big deal tho, it'll get fixed next time rails dumps the schema)